Cloud Application Manager/Troubleshooting/camdce-troubleshooting-tips.md

```diff
@@ -0,0 +1,289 @@
+
+**Cause 3**
+
+Providers and Deployment policy boxes ownership can only be transfered to other users or teams that belong to the same cost center. Each user/team have an assigned costcenter that cannot be changed.
+
+**Solution**
+
+Make sure the resource owner and the desired user to transfer ownership to belong to the same costcenter.
+
+### Instance fails to register in Cloud Application Manager
+
+**Cause**
+
+An instance registration (import) is waiting for the CAM agent to install, but the agent installation never seems to be completed.
+
+**Solution 1**
+
+Make sure the communication from the instance to be register to CAM appliance works correctly.
+
+1. Connect to the instance by SSH or RDP.
+
+2. Run a ping command to check basic connectivity:
+
+    ```
+    ping -c3 10.0.0.1
+    ```
+
+    **Note:** Replace 10.0.0.1 with your appliance IP or hostname.
+
+If the above command does not get a reply, please:
+
+1. Take the corrective actions inside the VM, routers and firewalls to allow unrestricted communication between the instance and the CAM appliance. Ensure that the port 443 is open for the CAM agent to connect back successfully to the CAM appliance.
+
+2. In Cloud Application Manager, open the instance details page and click **Retry import** if available, or **Cancel import** and start the import process again. The instance should now finish the registration process successfully.
+
+If the connectivity seems to be working as expected and the problem persists, connect to the instance over SSH or RDP and grab a copy of the agent `/var/log/elasticbox/elasticbox-agent.log` in the case of Linux or `ProgramData\ElasticBox\elasticbox-agent.log` in the case of Windows (please note that the folder is hidden in Windows and the exact path must be entered to navigate to the folder). Once the log file is collected please attach it when [Contacting support](mailto:incident@CenturyLink.com)
+
+**Solution 2**
+
+The CAM agent might be stopped and not able to complete the registration process due to the instance SSL stack not supporting TLS 1.2 at minimum. This might happen in old Windows machines using a .Net Framework version earlier than 4.5 (in the 4.5 version you have to opt-in to use it, while in versions 4.6 and above it is the default).
+
+Check if ElasticBox logs (`C:\program files\elastic Box` in Windows) have something such as:
+
+```
+2019-09-05 17:45:07Z Exception setting "SecurityProtocol": "Cannot convert null to type "System.Net.SecurityProtocolType" due to invalid enumeration values. Specify one of the following enumeration values and try again. The possible enumeration values are "Ssl3, Tls"."
+```
+
+This message confirms that the TLS 1.2 protocol required version is not supported in the instance. Upgrade the .Net Framework version for this issue to be addressed.
```

Comment:
This message confirms that the TLS 1.2 protocol required is not supported by the application executing the script.  Upgrade the application to a version that supports TLS 1.2 to address this issue.